src/components/App/index.tsx

```diff
@@ -122,13 +127,46 @@ export class AppBase extends React.Component<Props> {
     });
   }
 
+  dismissError = (errorId: number) => {
+    const { dispatch } = this.props;
+
+    dispatch(errorsActions.removeError({ id: errorId }));
+  };
+
+  renderErrors() {
+    const { errors } = this.props;
+
+    if (errors.length === 0) {
+      return null;
+    }
+
+    return this.renderRow(
+      <Col>
+        {errors.map((error) => (
+          <Alert
```

Comment:
We'll probably need to move this and the `onClose` logic to its own component but I suppose we could do that later

Reply:
I agree, thanks!